docs/config.json

```diff
@@ -617,6 +617,24 @@
             }
 	  ]
 	},
+	{
+	  "title": "Using Dynamic Configuration Resources",
```

Comment:
@ptgott this creates a duplicate section, see L607.

Reply:
Thanks! Must have missed that when resolving conflicts